src/JsonSchema/Constraints/Constraint.php

```diff
@@ -30,6 +30,7 @@
     const CHECK_MODE_COERCE_TYPES =     0x00000004;
     const CHECK_MODE_APPLY_DEFAULTS =   0x00000008;
     const CHECK_MODE_EXCEPTIONS =       0x00000010;
+    const CHECK_MODE_DISABLE_FORMAT =   0x00000020;
```

Comment:
Maybe this is just a personal style thing, and it's no big deal, but for some reason I always get a little uncomfortable when I see a variable or constant named in the negative. It makes it the odd duck out, and then you wind up doing odd double-negative logic to check them in the affirmative, eg `if (!disabledThing)`.

Is there any way to flip the polarity and enable it by default, then require the user to explicitly disable it if he doesn't want it?